stack.yaml

```diff
@@ -64,3 +64,7 @@ packages:
 #
 # Allow a newer minor version of GHC than the snapshot specifies
 # compiler-check: newer-minor
+
+nix:
+  packages:
+    - postgresql
```

Comment:
oops yeah left that in there, but does not hurt, now whoever runs stack with nix enabled gets postgresql automatically